Bitcoin.pdf: Add A Page That Lists Translations

harding opened the pull request:
Preview: http://dg2.dtrt.org/en/bitcoin-paper

This PR adds a new page to the site listing the original Bitcoin paper as well as all translations available on Bitcoin.org.  The PR also updates all existing direct links to the PDF to point to the new page so readers can discover what translations are available.

Note: this PR isn't complete yet---it requires @breathingdog's Spanish translation to be added
once the issues in #822 are resolved.  I'll rebase when that's available.

harding commented:
@breathingdog if you can provide a Spanish translation for the following text, I'll add it directly to the PR so it doesn't need to wait for our next Spanish update from Transifex:

    summary: "The paper that first introduced Bitcoin"
    description: "Satoshi Nakamoto's original paper is still recommended reading for anyone studying how Bitcoin works. Choose which translation of the paper you want to read:"
    translated_by: "translated by"
    submit_new_translation: 'Do you want to translate the paper into your language? Visit the <a href="https://github.com/saivann/bitcoinwhitepaper">Bitcoin Paper repository</a> for instructions and <a href="https://github.com/saivann/bitcoinwhitepaper/issues/new">open an issue</a> if you have any questions.'

breathingdog commented:
I could do a fast translation but always remembering I'm not a pro. The terms "translated_by" or "submit_new_translation" are commands? I suppose they shouldn't be translated, am I wrong?

harding commented:
@breathingdog if you want to leave the translation for the translators on Transifex, that's perfectly fine.  The terms before the first colon are keys, so they don't get translated.

breathingdog commented:
summary: "El documento que presentó por primera vez Bitcoin "
decription: "El documento original de Satoshi continúa siendo lectura recomendada para cualquiera que estudie cómo funciona Bitcoin. Escoge qué traducción del documento quieres leer:"
translated_by: "traducido por"
submit_new_translation: "¿Quieres traducir el documento a tu lengua? Para consultar las instrucciones, visita <a href="https://github.com/saivann/bitcoinwhitepaper">Bitcoin Paper repository</a> y si tienes preguntas, visita: <a href="https://github.com/saivann/bitcoinwhitepaper/issues/new">open an issue</a>"

breathingdog commented:
About the last line, it could be also this way, adding "el" before the link to the repository:

"¿Quieres traducir el documento a tu lengua? Para consultar las instrucciones, visita el <a href="https://github.com/saivann/bitcoinwhitepaper">Bitcoin Paper repository</a> y si tienes preguntas, visita: open an issue"

harding commented:
@breathingdog thanks!  I updated the PR and the preview:

* English: http://dg2.dtrt.org/en/bitcoin-paper
* Spanish: http://dg2.dtrt.org/es/bitcoin-documento

saivann commented on _plugins/translate.rb:
```diff
@@ -87,6 +87,12 @@ def render(context)
       end
+
+      ## Hack for renaming links to the Bitcoin paper. Safe to remove
+      ## when all langugages have "bitcoin-paper:" defined in the "url:"
```
@harding s/langugages/languages/

saivann commented:
@harding I suggest we use a single consistant directory for PDF documents, so either we keep them all at the root of the filesystem (/), or we move them all to files/bitcoin-whitepaper/ , and only keep a copy of the original English whitepaper at the root of the filesystem given how much backlink this paper currently has.

harding commented:
Several updates suggested by @saivann (thanks!):

* Moved `bitcoin_ru.pdf` and `bitcoin_es_latam.pdf` to `files/bitcoin-paper/` directory (with redirects)

* Virus scanned all three translated PDFs:

    - https://www.virustotal.com/en/file/1a5206ead919bb038d28321d33e1d753eaa32e61a011e7b21a10edb07273f73c/analysis/1429900417/
    - https://www.virustotal.com/en/file/96146715b9b6ffc46da155601da8ecbee37e2b4e71d6bffa45840d2a635d3683/analysis/1429901981/
    - https://www.virustotal.com/en/file/0b4dd87eb1e55004443d4fed4a9ba087c53bdfdce4bb9eb494eb37e690420f2b/analysis/

* Fixed typo

* Added comment to `_templates/bitcoin-paper` telling future paper authors/Bitcoin.org maintainers to generate their own PDFs and virus scan them.

The preview has been updated.

saivann commented:
@harding Thanks!

harding commented:
In the absence of critical feedback, I'll merge this PR around 23:00 UTC Saturday.
